src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -112,30 +122,67 @@ def on_time_step(self, event: Event):
         choose which households move;
         move those households to a new address
         """
-        households = self.population_view.subview(
-            ["household_id", "relation_to_household_head", "address", "zipcode"]
-        ).get(event.index)
+        households = self.population_view.get(event.index)
         household_heads = households.loc[
             households["relation_to_household_head"] == "Reference person"
         ]
         households_that_move = self.addresses.determine_if_moving(
             household_heads["household_id"], self.household_move_rate
         )
 
-        if len(households_that_move) > 0:
+        # Find households that move abroad and separate subsets of state table
+        households_that_move_aboard_idx = self.determine_households_moving_abroad(
+            households_that_move,
+            self.proportion_households_leaving_country,
+            event
+        )
+        moving_abroad_households = households_that_move.loc[households_that_move_aboard_idx]
+        moving_domestic_households = households_that_move.loc[
+            ~households_that_move.index.isin(moving_abroad_households.index)
+        ]
+        abroad_moving_households = households.loc[
```

Comment:
nit: i think you could just overwrite the `moving_abroad_households` dataframe with this one so that you aren't carrying around 2 dataframes with similar names. same for `domestic`

Reply:
I was looking at this when I implemented it.  One is a series and one is a dataframe -- A series of household_ids that move and then the dataframe is the state table with all members from those households which is what we use after that dataframe is created.